packages/near-membrane-base/src/membrane.ts

```diff
@@ -611,6 +644,34 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveObject(value: any): boolean {
+            if (value === null || value === undefined || value === ObjectProto) {
+                return false;
+            }
+            if (typeof value === 'object') {
+                if (
+                    // Arrays are marked as live in the BoundaryProxyHandler constructor.
+                    // isArrayOrNotOrThrowForRevoked(value) ||
```

Comment:
this is not related I believe, let's remove that... ArrayBugger.isView will never throw.

Reply:
It's a comment to show that we don't need to do the array check. It's not code to guard anything.

Reply:
Removed and updated comment to avoid confusion.